src/main/java/frc/robot/subsystems/PizzaBoxSubsystem/PizzaBoxConstants.java

```diff
@@ -12,4 +13,6 @@ public class PizzaBoxConstants {
     // important stuff end
 
     public final static double STOP = .00;
+    public final static double SERVO_NORM = .50; //this is what the servos go to angle wise when reseting. 0.50 is just an example
```

Comment:
the angle that the servos should go will need tuning anyways, especially when they are currently replacing the cape servo.

But for the pizza box motor that pushes the note out of the pizza box, neutral angle is 50, and the out angle is 180. Numbers taken from last season

Reply:
I will add that tomorrow because as a rule for my health I don't work past 9:30 pm